archey/test/test_archey_uptime.py

```diff
@@ -62,9 +63,89 @@ def test_day_and_hour_and_minutes(self):
         create=True
     )
     def test_days_and_minutes(self):
-        """Test when only days ABD minutes should be displayed"""
+        """Test when only days AND minutes should be displayed"""
         self.assertEqual(Uptime().value, '3 days and 3 minutes')
 
+    @patch(
+        'archey.entries.uptime.open',
+        side_effect=PermissionError(),
+        create=True
+    )
+    @patch(
+        'archey.entries.uptime.time.CLOCK_BOOTTIME',  # Ensure first `getattr` call succeeds anyhow.
+        create=True
+    )
+    @patch(
+        'archey.entries.uptime.time.clock_gettime',
+        return_value=1000,
+        create=True
+    )
+    def test_clock_fallback(self, _, __, ___):
+        """
+        Test when we can't access /proc/uptime on Linux/macOS/BSD.
+        We only test one clock as all clocks rely on the same builtin `time.clock_gettime` method.
+        """
+        self.assertEqual(Uptime().value, '16 minutes')
+
+    @patch('archey.entries.uptime.check_output')
+    def test_uptime_fallback(self, check_output_mock):
```

Comment:
Very clean test ! I've just removed the PyLint `pragma` as apparently unnecessary, and simplified it a bit with the corresponding method getting static.